src/ensembl/src/shared/components/radio/Radio.tsx

```diff
@@ -14,6 +16,9 @@ type Props = {
   classNames?: {
     radioInput?: string;
```

Comment:
I understand that this is from a previous commit, but we don't want to allow parent components to style the native radio input element at all — we always want to hide it. I think this field should be removed.